test/shared/src/main/scala/zio/test/environment/package.scala

```diff
@@ -783,6 +808,13 @@ package object environment extends PlatformSpecific {
     def debug[R <: TestConsole, E, A](zio: ZIO[R, E, A]): ZIO[R, E, A] =
       ZIO.accessM(_.get.debug(zio))
 
+    /**
+     * Accesses a `TestConsole` instance in the environment and returns the
+     * contents of the error buffer.
+     */
+    val errOutput: ZIO[TestConsole, Nothing, Vector[String]] =
```

Comment:
Update name to conform to change above.

Reply:
Sorry about missing that. Can you tell I'm a bit of a noob with this pattern? ;)

I've rebased on latest master and pushed a new version.

Reply:
No worries! You're contributing so you're already one step ahead! 😃